media/js/app/footprint-detail.js

```diff
@@ -613,6 +613,9 @@
             jQuery(this.elRecordkeeping).show();
         },
         maximizeCarousel: function(evt) {
+            if(jQuery('#carousel-modal').is(':visible')){
```

Comment:
hmmm...eslint isn't turned on here for some reason. Our format rules include some spacing, like so:
`if (jQuery('#carousel-modal').is(':visible')) {`

Reply:
Ah. Interesting. It gave me an error when it came to a semicolon and indentation but that's about it for linting. I'll look into it!